pika/adapters/blocking_connection.py

```diff
@@ -2010,7 +2013,7 @@ def basic_nack(self, delivery_tag=None, multiple=False, requeue=True):
                               requeue=requeue)
         self._flush_output()
 
-    def basic_get(self, queue=None, no_ack=False):
+    def basic_get(self, queue='', no_ack=False):
```

Comment:
What's the point of default value on the `queue` arg? Queue name is required, isn't it?